4/Step 3/ChicagoSprintShipper.ts

```diff
@@ -0,0 +1,9 @@
+import { Shipper } from "./Shipper";
+import { ShipmentCost } from "./AirEastShipper";
+
+export class ChicagoSprintShipper extends Shipper {
+    public standardPackageChange = 0.42;
+    public shipmentCost: ShipmentCost = {
+        letter: 0.42, package: 0.2, oversized: 0
```

Comment:
What are these explicit values? Shouldn't they be dynamic? You should calculate cost by weight.
`getCost(weight: number): number`